src/locales/de/translations.json

```diff
@@ -114,6 +114,8 @@
   "navigation": {
     "language-aria": "Sprachauswahl",
     "main-navigation-aria": "Hauptnavigation",
+    "en": "Englisch",
```

Comment:
I know this is the straight forward solution. Maintain a list of the languages for each available language. But I think we need to list the target language in it's own language. There is no point of translating `English` into the source language (here `Englisch`) because the user would not recognize the translated word anyway.

Similar to wikipedia where the language switch is always written in the target language:
![Screen Shot 2021-09-27 at 12 55 05](https://user-images.githubusercontent.com/1701755/134895656-941eea7d-ea4b-4ff8-8d38-c99aa52fe7a3.png)

```suggestion
    "en": "English",
```

Reply:
Nice catch, I changed it.